RELEASE_NOTES.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## stable-1.16-2.5.0, stable-1.17-2.5.0, stable-1.18-2.5.0
+
+* Cert-manager
+  - use `delete` upgrade-strategy.
+* Elasticsearch
```

Comment:
This should not be in the release. It's the PR we reverted. Can we turn off the automatic update for elasticsearch?

Reply:
Looking at milestone/2.5 it doesn't seem like elasticsearch is actually in there https://github.com/mesosphere/kubernetes-base-addons/milestone/7?closed=1

but other addons are like opsportal, nvidia and they aren't shown here

Reply:
Opsportal:
![image](https://user-images.githubusercontent.com/952828/96180475-4f8ff200-0ee7-11eb-8b3e-e5548f71b4c9.png)

Nvidia:
![image](https://user-images.githubusercontent.com/952828/96180530-6afafd00-0ee7-11eb-9cc4-0b831a9632a3.png)

Reply:
The Nvidia PR lists a COPS ticket that it resolves. Surely that should contain a release note as it directly fixes a customer problem?